pkg/apiserver/auth-config_test.go

```diff
@@ -39,158 +41,153 @@ import (
 	"kubevirt.io/containerized-data-importer/pkg/util"
 	"kubevirt.io/containerized-data-importer/pkg/util/cert"
 	"kubevirt.io/containerized-data-importer/pkg/util/cert/triple"
+	"kubevirt.io/containerized-data-importer/tests/reporters"
 )
 
-func generateCACert(t *testing.T) string {
+func TestAuthConfig(t *testing.T) {
```

Comment:
In general we try to have a single 'suite' file that has the 'Testing.T' in it. We name it: <package>_suite_test.go with the package here being apiserver. It also allows us to set things like logger in the before suite. A decent example is in pkg/controller/controller_suite_test.go